op-node/rollup/types.go

```diff
@@ -137,6 +137,8 @@ type Config struct {
 
 	// AltDAConfig. We are in the process of migrating to the AltDAConfig from these legacy top level values
 	AltDAConfig *AltDAConfig `json:"alt_da,omitempty"`
+
+	BlockBuildingThreshold time.Duration `json:"block_building_threshold,omitempty"`
```

Comment:
Is this a flag or part of the chainspec (rollup config)? Seems like its better as a flag?